doc/charm++/manual.rst

```diff
@@ -4835,7 +4834,6 @@ registered or pinned memory. Registered memory can be de-registered by calling
 the ``deregisterMem()`` method on the CkNcpyBuffer object.
 
 .. code-block:: c++
-
```

Comment:
The removal of this line seems like a typo

Reply:
Fixed